scipy/signal/signaltools.py

```diff
@@ -302,6 +262,73 @@ def correlate(in1, in2, mode='full', method='auto', return_lags=False):
                          " 'direct', or 'fft'.")
 
 
+def correlation_lags(in1, in2, mode='full'):
+    r"""
+    Calculate the lag/offset indices array for 1D cross-correlation.
+
+    Parameters
+    ----------
+    in1 : array_like
+        First input.
+    in2 : array_like
+        Second input. Should have the same number of dimensions as `in1`.
+    mode : str {'full', 'valid', 'same'}, optional
+        A string indicating the size of the output:
+
+        ``full``
+           The output is the full discrete linear cross-correlation
+           of the inputs. (Default)
+        ``valid``
+           The output consists only of those elements that do not
+           rely on the zero-padding. In 'valid' mode, either `in1` or `in2`
+           must be at least as large as the other in every dimension.
+        ``same``
+           The output is the same size as `in1`, centered
+           with respect to the 'full' output.
+
+    Returns
+    -------
+    lags : array
+        Returns an array containing cross-correlation lag/offset indices.
+        Indices can be indexed with the np.argmax of the correlation to return
+        the lag/offset.
+    """
```

Comment:
This docstring should be cross-referenced with `correlate`. It would also be good to have a notes section where you define  what texactlyhe lag is, in terms of the correlation equation.

Reply:
It would be good to have `correlation_lags` in the `See also` section of `correlate`.